### ISSUE TITLE
Could --analysis-mode scorpio make sense?

### ISSUE THREAD

**Issue by tseemann** [cov-lineages/pangolin#412]
I was wondering if it was possible to just run `scorpio` and skip USHER and PangoLearn altogether?

This would be for when one only cares about designated VOCs. It would be faster and still provide a pangolin compatible outfile.

But it's also possible I am not udnerstanding properly :-)

**Comment by aineniamh** [comment]
Yeah, I can add that in- theoretically the user could just run scorpio independently, but I guess it's useful to have the alignment step against the correct reference automated in pangolin.

**Comment by aineniamh** [comment]
I think I've got this in now, just checking tests pass. 
- The sequences will be classed as `Unassigned` if they don't get assigned something by scorpio
- If they do, the mrca_lineage field in scorpio will be reported
- The `version` column will be `SCORPIO_v0.1.4`, where v0.1.4 corresponds to the version of constellations that was used to do the scorpio assignments (I think reporting a Pango lineages version in this field would be misleading).

**Comment by tseemann** [comment]
Wow, thanks @aineniamh !
This will lower the barrier to using scorpio significantly.
eg.  `scorpio` command was not put into the `bin/` folder or PATH by default, and the reference setting etc.
